compiler/rustc_mir_build/src/thir/cx/expr.rs

```diff
@@ -939,9 +939,10 @@ impl<'tcx> Cx<'tcx> {
                 }
             }
 
-            // We encode uses of statics as a `*&STATIC` where the `&STATIC` part is
-            // a constant reference (or constant raw pointer for `static mut`) in MIR
+            // Writing a path to a static in Rust source creates a place expression AKA lvalue
+            // THIR calls it ExprKind::Deref, so here we gin up something to deref
```

Comment:
Honestly I find the new comment less useful than the old one. The new one doesn't tell me much about what happens and why.

Reply:
I don't find talking about MIR when implementing HIR -> THIR lowering to be useful.

Reply:
I also find the term "use" to be ambiguous. It means actually using its name by path.

Reply:
For instance, if you do this:
```rust
let x = &*STATIC;
x.called();
```
Is the second line a use of the STATIC? Arguably yes, if you are thinking in terms of use-def/def-use chains. But it's actually **completely irrelevant** for the lowering in question, which doesn't think about that.

Reply:
That all said, if you have a suggestion, I am happy to take it, but that's why I made the change I did to make it say much less and talk much more about local conditions.

Reply:
Ah, I see, you left two comments on slightly different line ranges.